include/openthread/netdiag.h

```diff
@@ -95,13 +95,14 @@ enum
     OT_NETWORK_DIAGNOSTIC_TLV_ANSWER              = 32, ///< Answer TLV
     OT_NETWORK_DIAGNOSTIC_TLV_QUERY_ID            = 33, ///< Query ID TLV
     OT_NETWORK_DIAGNOSTIC_TLV_MLE_COUNTERS        = 34, ///< MLE Counters TLV
-
+    OT_NETWORK_DIAGNOSTIC_TLV_VENDOR_APP_URL      = 35, ///< Vendor App URL TLV.
```

Comment:
nit: for consistency with above
```suggestion
    OT_NETWORK_DIAGNOSTIC_TLV_VENDOR_APP_URL      = 35, ///< Vendor App URL TLV
```